checker/src/main/java/org/checkerframework/checker/nullness/CollectionToArrayHeuristics.java

```diff
@@ -35,26 +43,48 @@
  *       </ol>
  * </ol>
  *
- * Note: The nullness of the returned array doesn't depend on the passed array nullness.
+ * <p>Additionally, when the lint option {@link NullnessChecker#LINT_TRUSTARRAYLENZERO} is provided,
+ * field accesses where the field declaration has a {@code @ArrayLen(0)} annotation are considered
```

Comment:
Can you please clarify this documentation?
 * I'm not sure where the field access will be.  Is the field access the receiver of `toArray`, or is the field access the argument to `toArray`?
 * Please be more specific than "are considered".  What is the behavior?

Reply:
I've reformulated that sentence.